docs/operators.md

```diff
@@ -1,68 +1,70 @@
-|    [ReduceSumSquare](https://github.com/onnx/onnx/blob/rel-1.2.3/docs/Operators.md#AReduceSumSquare)   |             |              |       :heavy_check_mark:        |
-|               [Relu](https://github.com/onnx/onnx/blob/rel-1.2.3/docs/Operators.md#Relu)               |     :heavy_check_mark:     |              |      :heavy_check_mark:      |
-|            [Reshape](https://github.com/onnx/onnx/blob/rel-1.2.3/docs/Operators.md#Reshape)            |     :heavy_check_mark:     |              |      :heavy_check_mark:      |
-|            [Sigmoid](https://github.com/onnx/onnx/blob/rel-1.2.3/docs/Operators.md#Sigmoid)            |     :heavy_check_mark:     |              |      :heavy_check_mark:      |
-|                [Sin](https://github.com/onnx/onnx/blob/rel-1.2.3/docs/Operators.md#Sin)                |     :heavy_check_mark:     |              |      :heavy_check_mark:      |
-|            [Slice](https://github.com/onnx/onnx/blob/rel-1.2.3/docs/Operators.md#Slice)            |     :heavy_check_mark:     |           |      :heavy_check_mark:      |
-|            [Softmax](https://github.com/onnx/onnx/blob/rel-1.2.3/docs/Operators.md#Softmax)            |     :heavy_check_mark:     |      :heavy_check_mark:     |      :heavy_check_mark:      |
-|              [Split](https://github.com/onnx/onnx/blob/rel-1.2.3/docs/Operators.md#Split)              |             |              |      :heavy_check_mark:      |
-|               [Sqrt](https://github.com/onnx/onnx/blob/rel-1.2.3/docs/Operators.md#Sqrt)               |     :heavy_check_mark:     |              |      :heavy_check_mark:      |
-|                [Sub](https://github.com/onnx/onnx/blob/rel-1.2.3/docs/Operators.md#Sub)                |     :heavy_check_mark:     |   :heavy_check_mark: |      :heavy_check_mark:      |
-|                [Sum](https://github.com/onnx/onnx/blob/rel-1.2.3/docs/Operators.md#Sum)                |     :heavy_check_mark:     |      :heavy_check_mark:     |      :heavy_check_mark:      |
-|                [Tan](https://github.com/onnx/onnx/blob/rel-1.2.3/docs/Operators.md#Tan)                |     :heavy_check_mark:     |              |      :heavy_check_mark:      |
-|               [Tanh](https://github.com/onnx/onnx/blob/rel-1.2.3/docs/Operators.md#Tanh)               |     :heavy_check_mark:     |              |      :heavy_check_mark:      |
-|               [Tile](https://github.com/onnx/onnx/blob/rel-1.2.3/docs/Operators.md#Tile)            |     :heavy_check_mark:     |           |     :heavy_check_mark:       |
-|          [Transpose](https://github.com/onnx/onnx/blob/rel-1.2.3/docs/Operators.md#Transpose)          |     :heavy_check_mark:     |              |      :heavy_check_mark:      |
-|          [Unsqueeze](https://github.com/onnx/onnx/blob/rel-1.2.3/docs/Operators.md#Unsqueeze)          |     :heavy_check_mark:     |              |     :heavy_check_mark:          |
```

Comment:
Not sure why the file diff is appearing like this. Something to do with line endings I think.  Just added two lines - Line 26 (Flatten) and Line 34 (InstanceNormalization). It previews correctly though.

Reply:
I guess this is because of EOL changed.